addons/base_automation/models/base_automation.py

```diff
@@ -376,124 +588,126 @@ def _register_hook(self):
         #
 
         def make_create():
-            """ Instanciate a create method that processes action rules. """
+            """ Instanciate a create method that processes automation rules. """
             @api.model_create_multi
             def create(self, vals_list, **kw):
-                # retrieve the action rules to possibly execute
-                actions = self.env['base.automation']._get_actions(self, ['on_create', 'on_create_or_write'])
-                if not actions:
+                # retrieve the automation rules to possibly execute
```

Comment:
I agree, I should have make this change in a preparation commit :-/

Reply:
done for `_get_actions` and the `__action_done` context var as these are rather central

also renamed the `_add_postmortem_*` method to be name-agnostic (no reason to have a long name, the short version is just as clear)

whilst i agree that a sed is sometimes noisy, i lived the opposite in the past with subscription in v9 where we renamed very little stuff to avoid 'polluting the diff' then years later you have nonsensical field names and variables who make the code way less readable, so sometimes a noisy PR is also a good thing compared to years of confusion 🙃

When balancing between diff pollution, retrocompat' and code-readability, imho diff pollution is the last important concern. I think that with this last change we have an acceptable balance between breaking stuff and having clear code.